java/core/src/java/org/apache/orc/impl/ConvertTreeReaderFactory.java

```diff
@@ -118,62 +121,62 @@ protected void assignStringGroupVectorEntry(BytesColumnVector bytesColVector,
      * length for the string group which can be (STRING, CHAR, VARCHAR).
      */
     protected void assignStringGroupVectorEntry(BytesColumnVector bytesColVector,
-        int elementNum, TypeDescription readerType, byte[] bytes, int start, int length) {
+                                                int elementNum,
+                                                TypeDescription readerType,
+                                                byte[] bytes,
+                                                int start,
+                                                int length) {
       switch (readerType.getCategory()) {
-      case STRING:
-        bytesColVector.setVal(elementNum, bytes, start, length);
-        break;
-      case CHAR:
-        {
+        case STRING:
+          bytesColVector.setVal(elementNum, bytes, start, length);
+          break;
+        case CHAR: {
           int adjustedDownLen =
               StringExpr.rightTrimAndTruncate(bytes, start, length, readerType.getMaxLength());
           bytesColVector.setVal(elementNum, bytes, start, adjustedDownLen);
+          break;
         }
-        break;
-      case VARCHAR:
-        {
+        case VARCHAR: {
           int adjustedDownLen =
               StringExpr.truncate(bytes, start, length, readerType.getMaxLength());
           bytesColVector.setVal(elementNum, bytes, start, adjustedDownLen);
+          break;
```

Comment:
ditto.

Reply:
Here I chose to move the break inside the brackets. Because if remove the {} . The context will show the problem of duplicate variables with the same name (`adjustedDownLen`) being defined.